src/output/style.rs

```diff
@@ -59,19 +59,19 @@ impl Format {
         &self,
         items: &[Item],
         globals: &mut dyn Scope,
-        file_context: &FileContext,
+        context: &impl FileContext,
```

Comment:
Now that the trait is named `FileContext`, I think the parameter can keep the old name `file_context`, making the change in this PR a few lines shorter (and fixing the format issue that the CI action complains about).

Also on lines 74,  365, and 400 in this file.

Reply:
I reverted it everywhere in this file, and fixed the formatting issues.